service/src/test/java/com/derpgroup/quip/manager/QuipManagerTest.java

```diff
@@ -72,4 +79,65 @@ public void testConversationRequest_another_noConversationHistory_noBot(){
     assertTrue(outputRawText.length() > 0);
     assertEquals(outputRawText,"I don't know how to handle requests for unnamed bots.");
   }
+  
+  @Test
+  public void testDetermineMaxQuipHistorySize(){
+    assertEquals(manager.determineMaxQuipHistorySize(16),8);
```

Comment:
All three of these test the same condition.  Change this input to be (max size * 2) + 1, and then check that the output is max size